tests/torch/quantization/test_algo_quantization.py

```diff
@@ -582,3 +582,30 @@ def test_shared_layers_are_weight_quantized_only_once():
     register_bn_adaptation_init_args(config)
     model, qctrl = create_compressed_model_and_algo_for_test(model, config)
     assert len(qctrl.weight_quantizers) == 1
+
+
+TEST_QUANTIZATION_PRESET_STRUCT = [
+    {
+        'preset': 'performance',
+        'expected_weights_q': SymmetricQuantizer,
+        'expected_activations_q': SymmetricQuantizer
+    },
+    {
+        'preset': 'mixed',
+        'expected_weights_q': SymmetricQuantizer,
+        'expected_activations_q': AsymmetricQuantizer
+    }]
+
+@pytest.mark.parametrize('data', TEST_QUANTIZATION_PRESET_STRUCT)
+def test_quantization_preset(data):
```

Comment:
Tests should be extended for various target devices and for the situation when there is both a preset and a specification of quantization for a given group/scope (via `scope_overrides`)

Reply:
Done.